proto/api/v1/process_listening_on_port_service.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+option go_package = "v1";
+option java_package = "io.stackrox.proto.api.v1";
+
+import weak "google/api/annotations.proto";
+import "storage/process_listening_on_port.proto";
+
+package v1;
+
+message GetProcessesListeningOnPortsByNamespaceRequest {
+    string namespace = 1;
+}
+
+message GetProcessesListeningOnPortsByNamespaceAndDeploymentRequest {
+    string namespace = 1;
+    string deployment_id = 2;
+}
+
+message ProcessListeningOnPortWithDeploymentId {
+    string deployment_id = 1;
+    repeated storage.ProcessListeningOnPort processes_listening_on_ports = 2;
+}
+
+message GetProcessesListeningOnPortsResponse {
+    repeated storage.ProcessListeningOnPort processes_listening_on_ports = 1;
+}
+
+message GetProcessesListeningOnPortsWithDeploymentResponse {
+    repeated ProcessListeningOnPortWithDeploymentId processes_listening_on_ports_with_deployment = 1;
+}
+
+service ProcessListeningOnPortService {
```

Comment:
What is the reason to separate this from process service?

Reply:
The endpoint information will be added to the process service at a future time. To best meet the criteria of the customer request in the MVP we decided to have a separate service.